mne/tests/test_annotations.py

```diff
@@ -658,7 +658,7 @@ def dummy_annotation_csv_file(tmpdir_factory):
     return fname
 
 
-@requires_pandas
+@requires_version('pandas', '0.16')
```

Comment:
Now it can happen that someone has pandas <0.16 installed, the `_check_pandas_installed(strict=True)` would work and it would get a weird message. 

I'm fine with letting it as it is, and wait for someone to complain in the issue tracker. 

The other option is to add a parameter `version` and raise a mining full error.

Reply:
true but I prefer this than bumping the versions for a so far niche usecase. I can live with the way it is now. We'll drop python 2 after the release anyway.